### DISCUSSION TITLE
ROWY projects just went blank

### DISCUSSION THREAD

**Discussion by st-trade** [AntlerVC/firetable#723] [Support / Q&A] [answered]
Please help!

My tables just disappeared from the rowy UI. Fortunately, all data in firestore is safe (functions, documents). I just can't access anything from rowy UI:

![image](https://user-images.githubusercontent.com/94030557/180101122-1e5d4dad-29e3-4abd-9f13-1c0c23028494.png)

**Comment by st-trade** [comment, marked as answer]
Fixed.

Apparently, one of my rowy projects was disconnected. I pasted again the cloud run URL, checked for updates, and it worked.
![image](https://user-images.githubusercontent.com/94030557/180105532-647e01a6-1d35-4d63-b5bf-255c0e21f75d.png)

**Reply by notsidney** [reply]
Hey there, if it happens again, could you try going to your_project_id.rowy.app/settings/debug and clicking **Reset local Firestore cache**?
<img width="251" alt="image" src="https://user-images.githubusercontent.com/27017118/180106082-57a0b9fa-0a62-4b70-9fff-a1c316ae1706.png">

We had a similar issue and found it was an error with the browser. I don’t know of any case where Rowy could become “disconnected”, since we store the table config in your Firestore.
